pkgpanda/util.py

```diff
@@ -450,7 +453,7 @@ def run(cmd, *args, **kwargs):
     if proc.returncode != 0:
         raise subprocess.CalledProcessError(proc.returncode, cmd)
 
-    assert len(stderr) == 0
+    # assert len(stderr) == 0
```

Comment:
I think we should add `-q  --quiet` or `-s --silent` option to disable logging and it should be on by default.